src/api/admin.ts

```diff
@@ -32,6 +32,13 @@ export type ListCellIdsResponse = Array<CellId>
 export type ListActiveAppsRequest = void
 export type ListActiveAppsResponse = Array<InstalledAppId>
 
+export type AgentInfoSigned = any
```

Comment:
instead of `any` here, why not `console.log` an instance of one of these and map out the type here?

Reply:
done